### PR TITLE
chore: make build work on Node 12

### PR DESCRIPTION
**Pull request by bcoe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "extend": "^3.0.2",
     "gaxios": "^5.0.1",
     "google-auth-library": "^8.0.2",
+    "linkinator": "^3.1.0",
```
Comment (alexander-fenster) [inline package.json:40]: It should be a devDep, should it?

Comment (bcoe) [inline package.json:40]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -69,14 +70,13 @@
     "karma-remap-coverage": "^0.1.5",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^5.0.0",
```
Comment (bcoe) [inline package.json:72]:
```suggestion
    "karma-webpack": "^5.0.0",
    "linkinator": "^3.1.0",
```

File: package.json (hunk 1 above)
Comment (bcoe) [reply package.json]: whoops, fixed.

**Review by alexander-fenster — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**